test/api/hateoas/HateoasLinksSpec.scala

```diff
@@ -16,9 +16,9 @@
 
 package api.hateoas
 
+import api.hateoas
```

Comment:
This line can be removed if we start replacing all the hateoas.Link with Link instead